sdk/sdk-view/src/test/java/mil/emp3/api/KMLExportTest.java

```diff
@@ -0,0 +1,675 @@
+package mil.emp3.api;
+
+import org.cmapi.primitives.GeoFillStyle;
+import org.cmapi.primitives.GeoIconStyle;
+import org.cmapi.primitives.GeoLabelStyle;
+import org.cmapi.primitives.GeoPosition;
+import org.cmapi.primitives.GeoStrokeStyle;
+import org.cmapi.primitives.IGeoAltitudeMode;
+import org.cmapi.primitives.IGeoRenderable;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.util.Random;
+
+import mil.emp3.api.abstracts.Feature;
+import mil.emp3.api.exceptions.EMP_Exception;
+import mil.emp3.api.interfaces.IEmpExportToStringCallback;
+import mil.emp3.api.interfaces.IFeature;
+import mil.emp3.api.interfaces.IMap;
+import mil.emp3.api.interfaces.IOverlay;
+import mil.emp3.api.utils.ComparisonUtils;
+import mil.emp3.api.utils.kml.EmpKMLExporter;
+
+import static mil.emp3.api.utils.ComparisonUtils.compareFeatureToPath;
+import static mil.emp3.api.utils.ComparisonUtils.compareFeatureToPolygon;
+import static mil.emp3.api.utils.ComparisonUtils.comparePath;
+import static mil.emp3.api.utils.ComparisonUtils.comparePoint;
+import static mil.emp3.api.utils.ComparisonUtils.comparePolygon;
+import static mil.emp3.api.utils.ComparisonUtils.compareTextToPoint;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+/**
+ * Created by matt.miller@rgi-corp.local on 11/15/17.
+ */
+
+public class KMLExportTest extends TestBaseSingleMap{
+    private final static String TAG = KMLExportTest.class.getName();
+    private static int count = 0;
+
+    @Before
+    public void setup() throws Exception{
+        super.init();
```

Comment:
super.init() was completely commented out last time I looked at it.

Reply:
it should be present